src/list.html

```diff
@@ -118,6 +118,10 @@ <h2 class="subtitle">
               <li><a href="https://nashdev.com" target="_blank">NashDev</a> - A network for Nashville developers.</li>
               <li><a href="https://knoxdevs.com" target="_blank">KnoxDev</a> - Knoxvilles Development Gathering </li>
             </ul>
+            <li>Philadelphia</li>
```

Comment:
Let's change this to Pennsylvania and wrap it in `<h3>` tags.
Also, move all the new code to after Oregon to keep it in alphabetical order.